structured_type.go

```diff
@@ -75,9 +75,19 @@ type StructuredObjectWriterContext interface {
 	WriteNullTime(fieldName string, value sql.NullTime, tsmode []byte) error
 	WriteStruct(fieldName string, value StructuredObjectWriter) error
 	WriteNullableStruct(fieldName string, value StructuredObjectWriter, typ reflect.Type) error
+	// WriteRaw is used for inserting slices and maps only.
+	WriteRaw(fieldName string, value any, tsmode ...[]byte) error
```

Comment:
if only slices and maps, can we type this properly and not use "any"?

Reply:
No, we can't. Generics are not supported in interfaces. Go rulez...